docs/components/databases.rst

```diff
@@ -6,6 +6,7 @@ Why do we use "databases"?
 
 We use databases to store data while we run the tests. When globally talking
 about databases, we are indirectly talking about the following subsystems.
+(JSON)
```

Comment:
Where does this come from?

Reply:
Line break. (The 80 chars for consoles support)

I added it as this part is about JSON and does have nothing to do with the SQL layout